app/javascript/controllers/scroll_to_top_controller.js

```diff
@@ -0,0 +1,13 @@
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+  static targets = [ 'scrollToTopBtn' ]
+
+  scrollToTop() {
```

Comment:
Maybe we can rename this action to `call()` or something like that, `scrollToTop` feels redundant to me looking at the controller name